patches/mongodb-query-parser+1.4.3.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/node_modules/mongodb-query-parser/lib/index.js b/node_modules/mongodb-query-parser/lib/index.js
```

Comment:
Did you intend to leave this file checked in?

Reply:
@JLLeitschuh yep, have to as query-parser was missing support for the $db parameter, and Timestamp was using the new bson format, while the dependency was still using the old format, so needed to make sure they were consistent